CreeDictionary/CreeDictionary/settings.py

```diff
@@ -161,7 +161,17 @@
     STATIC_URL = "/cree-dictionary/static/"
 
 STATIC_ROOT = os.path.join(BASE_DIR, "static")
-STATICFILES_STORAGE = "django.contrib.staticfiles.storage.ManifestStaticFilesStorage"
+
+
+if DEBUG:
+    # Use the default static storage backed for debug purposes.
+    STATICFILES_STORAGE = "django.contrib.staticfiles.storage.StaticFilesStorage"
+else:
+    # In production, use a manifest to encourage aggressive caching
+    # Note requires `python manage.py collectstatic`!
+    STATICFILES_STORAGE = (
+        "django.contrib.staticfiles.storage.ManifestStaticFilesStorage"
```

Comment:
This storage backend _requires_ `collectstatic` to be run before the URL reversing can work. Doing this avoids a crash when testing.